samples/boards/frdm_kl25z/accelerometer/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Add the sample in a separate commit. Also, put it in `samples/sensor/mma8451q`.

Reply:
sample will be in a separate PR